src/control/lib/control/pool_property.go

```diff
@@ -19,6 +19,12 @@ import (
 	"github.com/daos-stack/daos/src/control/drpc"
 )
 
+/*
+#include <daos/object.h>
+#include <daos/cont_props.h>
+*/
+import "C"
+
```

Comment:
We have been trying to keep uses of cgo confined to specific packages rather than sprinkling it around the codebase. Partially for aesthetic reasons, and partially to make it easier to debug build problems. The `daos` tool is an exception because it is tightly-coupled to the client library.

I think it would probably be best to create helpers in [drpc/property.go](https://github.com/daos-stack/daos/blob/master/src/control/drpc/property.go), e.g. `EcCellSizeValid(sz uint64) bool`.